_docs/services/app-autoscaler.md

```diff
@@ -0,0 +1,472 @@
+`responsetime` | "ms" (milliseconds) | The average amount of time the application takes to respond to a request in a given time period
+`throughput` | "rps" (requests per second) | The total number of the processed requests in a given time period
+custom | User Provided | You can define your own metric name and emit the metric to App-AutoScaler to trigger dynamic scaling. More on this is below.
+
+
+## Basic Usage
+
+Let's assume there is an application which needs to be scaled up to two application instances if the disk utilization goes above 60% and scale back down if goes below that value. Using the App-Autoscaler only requires a few steps to use:
+
+ - Push an app
+ - Create a Service Instance of App-AutoScaler
+ - Create Scaling Policy JSON
+ - Bind the Service Instance to the application and provide the Scaling Policy JSON
+
+```bash
+# Push a copy of the application
+cf push my_app
+
+# Create a service instance of app-autoscaler called "my-autoscaler-service-instance"
+cf create-service app-autoscaler autoscaler-free-plan my-autoscaler-service-instance -b autoscaler 
+
+# Bind the service instance to the application and specify the Scaling Policy
+cf bind-service my_app my-autoscaler-service-instance -c '{"instance_min_count":1,"instance_max_count":2,"scaling_rules":[{"metric_type":"diskutil","breach_duration_secs":60,"threshold":60,"operator":">=","cool_down_secs":60,"adjustment":"+1"},{"metric_type":"diskutil","breach_duration_secs":60,"threshold":30,"operator":"<","cool_down_secs":60,"adjustment":"-1"}]}'
+```
+
+App-AutoScaler will monitor the application at ~40s intervals and scale the application instances based on the Scaling Policy defined.
+
+## Creating a Scaling Policy
+
+The Scaling Policy defines the rules of how, why and when to scale application instances in JSON format.  Using the example from the previous section, a more human readable format would be:
+
+
+```JSON
+{
+    "instance_min_count":1,        
+    "instance_max_count":2,
+    "scaling_rules":
+    [
+        {
+            "metric_type":"diskutil",
+            "threshold":60,
+            "operator":">=",
+            "adjustment":"+1",
+            "breach_duration_secs":60,
+            "cool_down_secs":60
+        },
+        {
+            "metric_type":"diskutil",
+            "threshold":30,
+            "operator":"<",
+            "adjustment":"-1",
+            "breach_duration_secs":60,
+            "cool_down_secs":60
+        }
+    ]
+}
+```
+
+The top level keys used to define a Scaling Policy are:
+
+ - `instance_min_count` - The minimum number of application instances that can be scaled down to. This integer value cannot be less than 1. This key/value is required.
+ - `instance_max_count` - The maximum number of application instances that can be scaled up to.  This integer value has no upper boundary but be greater than the organization quota for memory allows for at the time of scaling.  This key/value is required.
```

Comment:
typo or structure issue: ...upper boundary (may/must/??) be greater than the organization quota...